soda_machine.rb

```diff
@@ -1,18 +1,36 @@
 class SodaMachine
   attr_reader :sodas, :cash
 
+#args = {sodas: [brand, price],
+#        cash: price
+#         }
+
   def initialize(args = {})
     @sodas = args[:sodas]
     @cash = args[:cash]
   end
 
   def current_inventory_count
+    @sodas.length
   end
 
+#access the soda from the array in which brand = soda_branch
+#@sodas includes instance variables of each soda stored in an array
   def find_soda(soda_brand)
+    @sodas.find {|soda_name| soda_name.brand == soda_brand}
   end
 
+#remove from sodas array, and adds price from cash
   def sell(soda_brand)
+    @sodas.delete_if {|soda| soda.brand == soda_brand}
+    @cash += 1.00
```

Comment:
cash should add the price of the soda instance, not just 1.00, right?  what if a soda is 1.50 or some other price?